packages/core/src/types.ts

```diff
@@ -16,10 +16,24 @@ export const enum KeyDerivationMethod {
   Raw = 'RAW',
 }
 
+export enum WalletScheme {
+  DatabasePerWallet = 'DatabasePerWallet',
+  MultiWalletSingleTable = 'MultiWalletSingleTable',
+  MultiWalletSingleTableSharedPool = 'MultiWalletSingleTableSharedPool',
+}
```

Comment:
This is  a postgres wallet feature, so maybe we can also move this to the node package?